src/pint/utils.py

```diff
@@ -3,6 +3,11 @@
 from contextlib import contextmanager
 from copy import deepcopy
 from io import StringIO
+import getpass
```

Comment:
It'd be good to run `isort` over this file to put the imports in standard order and format.